fastai/vision/learner.py

```diff
@@ -114,6 +114,18 @@ def _cl_int_plot_top_losses(self, k, largest=True, figsize=(12,12)):
         cl = int(cl)
         im.show(ax=axes.flat[i], title=
             f'{classes[self.pred_class[idx]]}/{classes[cl]} / {self.losses[idx]:.2f} / {self.probs[idx][cl]:.2f}')
+        if(heatmap):
```

Comment:
No parenthesis, just heatmap (style guide).

Reply:
This is fixed.